Themes/default/css/index.css

```diff
@@ -57,6 +58,10 @@ textarea, textarea.editor {
 input, button, select {
 	font-size: 0.9em;
 }
+/* Height fixation */
+input, select {
```

Comment:
I changed stuff but probably missing some more lists around if you find any you can use "select_list" class to fix them.